spec/src/main/asciidoc/architecture.asciidoc

```diff
@@ -35,11 +35,10 @@ The specifications for each release are listed in reverse chronological order.
 [[microprofile6.0]]
 === MicroProfile 6.0
 
-MicroProfile 6.0 replaces required individual Jakarta specifications in prior releases like Jakarta Restful Web Services and JSON Binding with the Jakarta EE 10 Core Profile.
 
-Based on MicroProfile's time-boxed release process, this is a major release that includes backward incompatible changes. This release requires Jakarta EE Core Profile 10, which uses the `jakarta` namespace introduced in Jakarta EE 9.
+MicroProfile 6.0 adds support for the Jakarta EE 10 Core Profile as an alternative to supporting individual Jakarta specifications like Jakarta Restful Web Services and JSON Binding as in prior releases.  Additionally, 6.0 adds requirements for all required Jakarta TCKs to be passed.
```

Comment:
The phrase:

> Additionally, 6.0 adds requirements for all required Jakarta TCKs to be passed.

seems redundant since that requirement was already there.

Reply:
Do you mean it was already there in other sections that mention 6.0 or was already there in 5.0?

Reply:
I mean there was a long discussion a few years back about whether or not a MicroProfile implementation required the CDI/JAX-RS and JSON-* specifications to pass and my understanding of the conclusion of that discussion was that they were. Making it clearer in the text that this is required is obviously a good thing, but stating that you are adding a requirement that was already there reverses the outcome of the discussion we had in the past about CDI/JAX-RS and JSON-* TCKs needed to be passing.

Reply:
@NottyCode it was the reverse.  Prior to [this PR](https://github.com/eclipse/microprofile/commit/ce9ad44853d22412238559bcd5e35cebca740631) from John in October the spec said, "Passing the Jakarta specifications TCKs is optional."  I was attempting to be consistent with that PR and state that if you are on Jakarta EE 9.1, passing the related Jakarta TCKs is still required.

Reply:
In the past, it was argued that CDI TCKs contained EJB, which was not supported by MicroProfile. The whole point of Jakarta EE 10 Core Profile was to fix the problem so that Jakarta EE 10 Core Profile can be consumed by MicroProfile and strengthen up the compliance, which was why MP adopts Jakarta EE 10 Core Profile.

Reply:
We are ok passing all related Jakarta EE 9.1 TCK tests.